### ISSUE TITLE
Add a user-friendly documentation about all available customization options

### ISSUE THREAD

**Issue by philipgiuliani** [videojs/video.js#2730]
I couldn't find any docs about customizing the player. For example i know (because of a PR) that it's possible to change the volume slider so that it is vertical.

But where can i find this option, and all other available options?

**Comment by gkatsev** [comment]
I'm not sure we're doing the best we can to document these things. We do have (probably better docs than we used to have) some docs on the site: http://docs.videojs.com/
If you have the time and can help with documentation, that would be amazing.

**Comment by leventmuslim** [comment]
Allow me revive this old topic... 
Sadly I agree on Philip on this

I'm not very experienced with coding myself, I'm more of a GUI person. Although, as I read on the present documentations, I am beginning to see that the initial intention is to focus on improving the Videojs project itself instead of working on side-workloads like updating docs frequently. 
Thus, docs are universal, instead of detailed; ie: they don't say put this file into this folder.  (-: Oh gee how I wish they were so! 
And to be honest official docs are everywhere and easy to reach, whether on github or on its own dedicated website, and so on.

Then again.. Sometimes it can get really difficult to find out in what order things must be done to have it done properly. For example,even a considerably easy to do action like "changing the players language" can get tricky.

This can be overcome for less experienced users like me.. and others! 
I'm not very sure on this one but, how about some video snippets about how you make things work? Because writing explanations to documents aren't easy at all either. Docs supported with short video examples would be great. I would read the piece of document I need, then watch it live and take the same steps with a better and more confident understanding. 
Is this doable?

I mean.. after all Videojs is a project for videos, right? So, why plain text docs only?

**Comment by at-cuongtran** [comment]
I also agree with Philip, I'm a web front end developer and I have worked with several frameworks and libraries... But I still find this document is hard to follow

**Comment by gkatsev** [comment]
We agree that our documentation is severely lacking. We'd love to improve it and unfortunately, it always gets de-prioritized compared to other work. The easiest thing we can do is probably de-emphasize our generated jsdocs and focus on our hand-written guides, which, while better, they definitely have room for improvement.
If you have a suggestion on a particular thing that you find lacking, please let us know.

As for making a video example, that's something I'd love to do but the time investment compared to plain text docs is enormous. Given our time constraint, unfortunately, we won't be able to do it in the near future.
